sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -38,6 +37,16 @@
  */
 public class SensitivityAnalysisResult {
 
+    /*
+     * Key type for function reference value when stored function type, contingency and function id
+     */
+    private class FunctionReferenceKey extends Triple<SensitivityFunctionType, String, String> {
```

Comment:
Could you do also fully define this object?

Reply:
Why ?

Reply:
Be careful about equals/hashcode and inheritance:
What is the expected behaviour of ?:
```java
        assertEquals(new SensitivityAnalysisResult.FunctionReferenceKey(SensitivityFunctionType.BRANCH_ACTIVE_POWER_1, "dd", "xx"),
                new Triple<>(SensitivityFunctionType.BRANCH_ACTIVE_POWER_1, "dd", "xx"));
```
I would prefer having a dedicated object too, or at least use the raw Triple.

Reply:
Defining a new object will force us to implement new useless unit tests to satisfy coverage threshold. I'd rather go back to raw Triple then.

Reply:
Ok let's do that for function key and keep the dedicated object for variable as there is no "Quad" object.